exercises/pig-latin/.meta/src/reference/java/PigLatinTranslator.java

```diff
@@ -21,14 +21,12 @@ String translate(String sentence) {
                 .collect(Collectors.joining(" "));
     }
 
-    private static int yLocation;
-
     private String translateWord(String word) {
         if (wordStartsWithVowelLike(word)) {
             return word + AY;
         }
 
-        if(hasYAfterConsonantCluster(word)){
+        if (getLocationOfYAfterConsonantCluster(word) != -1) {
```

Comment:
You need to store the result of this method call in a variable and use it inside the `if` statement. Right now the build is failing because you're using the variable `yLocation` which doesn't exist anymore 🙂

Reply:
Oops. Thanks for catching that one.